client/src/pages/AnnouncementPage.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Sidebar from "../components/Sidebar";
+import SelectDropDownComponent from "../components/Dropdown";
+
+const AnnouncementPage = () => {
+  return (
+    <div className="flex w-full">
+      <Sidebar />
+      <div className="container w-full">
+        <h1 className="font-bold p-3 text-lg">Announcements</h1>
+        <hr></hr>
+        <div class="block p-6 rounded-lg shadow-lg bg-white w-full">
+          <div className="flex">
+            <img
+              src="https://mdbcdn.b-cdn.net/img/new/avatars/2.webp"
+              className="rounded-full w-12"
+              alt="Avatar"
+            />
+            <h5 class="leading-tight font-bold ml-3 mb-2">Admin</h5>
+            <p class="leading-tight ml-2 mb-3 text-xs">
+              Sept. 21, 2022 at 12:44PM
+            </p>
+          </div>
```

Comment:
- [ ] add the `Thread` button that appears at the top-right corner of the card onHover of `AnnouncementCard`

Reply:
I was thinking nga we can have the whole thread task separate from this task. Should we add it na?

Reply:
Yes. Kindly add it in for this task but just the thread button onHover. No functions needed.